src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/domain/Email.java

```diff
@@ -0,0 +1,51 @@
+package pl.tomaszdziurko.jvm_bloggers.mailing.domain;
+
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "email")
+@Data
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class Email {
+
+    @Id
+    @GeneratedValue(generator = "EMAIL_SEQ", strategy = GenerationType.SEQUENCE)
+    @SequenceGenerator(name = "EMAIL_SEQ", sequenceName = "EMAIL_SEQ",
+        allocationSize = 1)
+    private Long id;
+
+    @Column(name = "from_address", nullable = false, length = 250)
```

Comment:
We could avoid `@Column` annotation by using `org.hibernate.cfg.ImprovedNamingStrategy` which converts CamelCase variable to camel_case column name. For more info please see http://stackoverflow.com/questions/5243711/generated-column-and-table-names-in-hibernate-with-underscore